.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+before_install:
+  - gem install bundler
```

Comment:
@whomwah I am not sure what the `cache` key does but I tend not to specify the rest (such as RVM versions) unless those are really necessary. I think it's better to leave it to Travis to decide what the default version of Ruby should be and test using that.

Obviously in case of needing to test this gem on multiple versions of Ruby, that becomes essential.

The `before_install` step is there to update the version of Bundler that's currently installed on Travis. See the commit message of https://github.com/liquid/ruby-mpd/commit/7d63584028e3b31a5db9cba90608710ca4f40933